### DISCUSSION TITLE
Dot after name

### DISCUSSION THREAD

**Discussion by ayushsgithub** [vercel/next.js#56238] [Help]
### Summary

I don't know why but there's a dot appearing after my name when I'm trying to book my next.js conf ticket, I tried to edit but it's still there
![Screenshot 2023-09-30 173801](https://github.com/vercel/next.js/assets/120788538/f55b2e55-fa36-42dc-9aab-daeb213a17b5)

**Comment by leerob** [comment]
This is part of the design, it cannot be removed, but you can customize the rest of your name and/or title!

**Reply by ayushsgithub** [reply]
Oh, now I get it, thank you @leerob, you're an inspiration and I'm a huge fan
